scripts/install_third_party.sh

```diff
@@ -23,6 +23,8 @@ $PYTHON_CMD scripts/install_third_party.py
 
 # Install third-party node modules needed for the build process.
 install_node_module ajv 5.0.0
+install_node_module browserstack-local
```

Comment:
This and the following line should have version numbers.

Reply:
Done, thanks!